qa/suites/basic/health-ok.sh

```diff
@@ -81,22 +86,12 @@ ceph_log_grep_enoent_eaccess
 test_systemd_ceph_osd_target_wants
 rados_write_test
 ceph_version_test
-if [ -z "$MINI" ] ; then
-    run_stage_0 "$CLI"
-    restart_services
-    mon_restarted "1" # 1 means not restarted
-    osd_restarted "1"
-    # apply config change
-    change_osd_conf
-    change_mon_conf
-    # construct and spread config
-    run_stage_3 "$CLI"
-    restart_services
-    mon_restarted "0" # 0 means restarted
-    osd_restarted "0"
-    # make sure still in HEALTH_OK
-    ceph_cluster_status
-    ceph_health_test
```

Comment:
@jschmid1 Are you OK to drop this test of the restart functionality? Maybe the test should be re-implemented under `ceph.smoketests` before we really drop it?

Reply:
Getting rid of it now means that we have at least for a short period of time no restart tests, right?
In that case I'd keep them in for now and remove them _after_ we implemented the restart.functests